src/components/NewWorkflow/AddPhase/AddTask.js

```diff
@@ -23,37 +24,105 @@ class AddTask extends Component {
-        });
-        this.props.dispatch({type: 'ADD_TASK', payload: this.state})
-    }
+        Swal.fire({
+            title: 'Add a new task to curent phase',
+            input: 'text',
+            showCancelButton: true,
+            confirmButtonColor: '#014AAD',
+            cancelButtonColor: '#EC1515',
+            confirmButtonText: 'Next &rarr;',
+          }).then((result) => {
+            if (result.value) {
+              Swal.fire({
+                title: result.value,
+                text: "add optional steps to task",
+                input: 'select',
+                inputOptions: {
+                    nothing: 'no options needed',
+                    button: 'click a button',
+                    checkbox: 'add a checkbox',
+                    radio: 'select one option from list',
+                    email: 'enter an email',
+                    url: 'enter a url'
+                  },
+                inputPlaceholder: 'Select an option',
+                showCancelButton: true,
+                confirmButtonColor: '#014AAD',
+                cancelButtonColor: '#EC1515',
+              }).then((option) => {
+                if (option) {
+                    Swal.fire({
+                        title: 'keept this task?',
+                        text: `${result.value} with ${option.value}`,
+                        showCancelButton: true,
+                        confirmButtonColor: '#014AAD',
+                        cancelButtonColor: '#EC1515',
+                        confirmButtonText: 'keep'
+                    })}
+                    return(option)
+                }).then((keep) => {
+                    if (keep.value) {
+                        this.displayTask.push({post: {
+                            taskName: result.value,
+                            taskOption: keep.value,
+                            time: new Date()
+                        }})
+                        this.setState({
+                            showdata: this.displayTask,
+                            post: [...this.state.post, {
+                                taskName: result.value,
+                                taskOption: keep.value,
+                                time: new Date()
+                            }]
+                     });
+                }})
+            }
+          })
+
 
-    //this is not the appropriate data to publish full workflow
-    publish=()=>{
-        this.props.dispatch({type: 'PUBLISH_TASK', payload: this.displayTask})
     }
 
+    // //this is not the appropriate data to publish full workflow
+    // publish=()=>{
+    //     this.props.dispatch({type: 'PUBLISH_TASK', payload: this.displayTask})
+    // }
+
     render() {
+        console.log(this.displayTask)
         return (
             <div data-id={this.props.data} className="taskCard">
                 <hr/>
-                <form onSubmit={this.handleSubmit}>
+                {/* <form onSubmit={this.handleSubmit}>
```

Comment:
is this supposed to be commented out?

Reply:
yes, its a work in progress between collecting info and displaying it...